### DISCUSSION TITLE
Why were the patch versions for CVE-2022-35957, CVE-2022-21702, CVE-2022-21713 released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [grafana/grafana#78195] [General]
We are a research team dedicated to Golang, have discovered that CVE-2022-35957, CVE-2022-21702, CVE-2022-21713 were addressed in commit 3282afc648edc6cd5793056753b2ffc397c859aa, 605d056136b3dce5545bdbd7868acd8778eed9b1, 605d056136b3dce5545bdbd7868acd8778eed9b1. However, upon analyzing the commit, we observed that the patch version (v8.5.15, v8.5.0-beta1, v8.5.0-beta1) were released after a lapse of over one month. We are interested in understanding the reasons behind this delay in releasing the patch version, as it could potentially impede the prompt dissemination of patches to downstream users. We seek clarification on whether the delay might be attributed to:

1. Issues with testing and CI checking.
2. Other commits requiring inclusion in a single release.
3. By convention, infrequent release of versions.
4. Other reasons.
   We appreciate your attention to this matter and eagerly await your response. Thank you.

**Comment by natellium** [comment]
Hey there, apologies we didn't respond before. We're in the process of moving to GH issues. You can read more about this in https://github.com/grafana/grafana/discussions/73424.
Regarding your question, we don't understand what you're referring to. All the referenced CVEs are for Grafana specific:
- CVE-2022-35957, fixed for 9.1.6 & 8.5.13
- CVE-2022-21702, fixed in 8.3.6
- CVE-2022-21713,  fixed in 8.3.6

8.5.15  was just a Go version bump.

I will be closing this discussion, but if this is still a concern, please open an [issue](https://github.com/grafana/grafana/issues/new/choose) with additional details. Thanks for your contributions!
